src/main/java/com/google/sps/data/Post.java

```diff
@@ -43,9 +63,35 @@
   private String description = "";
   private String collegeId = "";
   private int timeSort = 0;
+  private int rank = 0;
 
   public boolean valid = true; // If false, the post shouldn't be saved - it might have malicious data.
 
+  /* Constructor for Post testing */
+  public Post(String organizationName, int month, int day, int startHour, 
```

Comment:
Is this the expected output of the Java formatter?

Reply:
I removed the constructor!